src/main/resources/static/js/Inscription_mentor.js

```diff
@@ -33,8 +31,7 @@ function submit(event) {
             break;
         }
 
-        if (i == 3 && Date.parse(inputs[i].value) < Date.now()) {
-            console.log(Date.parse(inputs[i].value) + " " + Date.now());
+        if (i == 3 && (new Date() >= new Date(inputs[i].value))) {
```

Comment:
THis message "Date Incorrect " means nothing for the end user cause even there the user does not know why the date is incorrect or what s(he) must do to avoid incorrectness.

Put something like "Veuillez entrer disponibilité à partir de demain"